langchain/package.json

```diff
@@ -1101,6 +1105,7 @@
     "node-llama-cpp": "*",
     "notion-to-md": "^3.1.0",
     "officeparser": "^4.0.4",
+    "open": "9.1.0",
```

Comment:
Would really love to not use this - any workaround?

Reply:
OK, I used console log instead to print the url for manually open